manim/utils/ipython_magic.py

```diff
@@ -88,19 +195,9 @@ def construct(self):
 
                 file_type = mimetypes.guess_type(config["output_file"])[0]
                 if file_type.startswith("image"):
-                    display(Image(filename=config["output_file"]))
-                    return
-
-                # videos need to be embedded when running in google colab
-                video_embed = "google.colab" in str(get_ipython())
-
-                display(
-                    Video(
-                        tmpfile,
-                        html_attributes=f'controls autoplay loop style="max-width: {config["media_width"]};"',
-                        embed=video_embed,
-                    )
-                )
+                    display(image_viewer(tmpfile, "350px", "900px"))
```

Comment:
I've set "350px", "900px" after some testing and found that these are nice sizes.
But maybe this can also be set by the `media_width` flag that was introduced here:
#1074

Reply:
https://ipywidgets.readthedocs.io/en/latest/examples/Layout%20Templates.html?highlight=GridspecLayout#Style-attributes

Since both Video/Image are using GridspecLayout, based on the ipythonwidget documentation this seems like you should be passing in the config.media_width instead of a hard coded value since the values there are responsive

Reply:
Yes, that is also my idea to pass these values here.
However, `config.media_width` is one value, and we would have the small size and the large size. So should  media_width be then a tuple?

Reply:
I think if the height is set to `100%` then it'll be capped on the width (e.g. if `width = 50vw` the larger video won't take up the full height but be limited by the width). But I think the thumbnail/smaller one is okay to be a smaller dynamic size like 20% width or something.

Reply:
Sounds reasonable.
Currently, I have other coding priorities, so I won't be able right now to implement this feature. My preferred strategy would be to merge this soon (has the hard-coded pixel values are working properly) and iterate over it in a future pr. Or if you have time and are interested, feel free to push a commit on this.

Reply:
I'll take a look and commit a few things